views/admin/admin/display.php

```diff
@@ -0,0 +1,13 @@
+<?php echo head(array(
+    'title' => __('Elasticsearch | Display Configuration')
```

Comment:
@jaguillette I tried toggling on and off the timestamps via the link you provided but I couldn't see where on the different pages(exhibit, neatline, simple pages) those new fields were being displayed. Could you point me to where they are shown?

Otherwise everything looks good!

Reply:
Oh, I should have mentioned that! The timestamps appear in the search results when the setting is enabled, and don't when it isn't. You have to search for something to see those different content types display. I usually search for "test"